default/stringtables/en.txt

```diff
@@ -7493,7 +7493,9 @@ SITREP_NOVA_DETECTED_LABEL
 Supernova detected
 
 SITREP_BUREAUCRACY_STOPPED
-The expanding [[policy PLC_BUREAUCRACY]] has halted work due to insufficient administrative facilities. Of [[buildingtype BLD_REGIONAL_ADMIN]] and [[buildingtype BLD_IMPERIAL_PALACE]], currently a total of %rawtext:havebuildings% are owned, but %rawtext:needbuildings% are needed to improve planet stability by [[value PLC_BUREAUCRACY_STABILITY_FLAT]]. [[policy PLC_BUREAUCRACY]] requires an additional administrative building every [[value BUREACRACY_ADMINS_NEEDED_TURN_SCALING]] turns.
+'''The [[policy PLC_BUREAUCRACY]] is expanging, requiring more administrative facilities.
+Next turn of [[buildingtype BLD_REGIONAL_ADMIN]] and [[buildingtype BLD_IMPERIAL_PALACE]] it will need %rawtext:needbuildings% or it will halt, currently a total of %rawtext:havebuildings% are owned.
```

Comment:
```suggestion
Starting next turn, %rawtext:needbuildings% [[buildingtype BLD_REGIONAL_ADMIN]] and [[buildingtype BLD_IMPERIAL_PALACE]] will be needed to prevent the the bureaucracy from halting. Currently, a total of %rawtext:havebuildings% are owned.
```